README.md

```diff
@@ -622,61 +622,61 @@ By index (be careful with the range):
 Resource re = rs.member(0);
 ```
 
-* Get the attachment set of this work order
+* Get the attachment set of this work order:
 
 ```java
 AttachmentSet ats = re.attachmentSet();
 ```
 
-* Initial the document data
+* Initial the document data:
 
 ```java
 String str = "hello world @ "+ Calendar.getInstance().getTime().toString();
 byte[] data = str.getBytes("utf-8");
 ```
 
-* Initial the attachment files
+* Initial the attachment files:
 
 ```java
 Attachment att = new Attachment().name("attachment.txt").description("test").data(data).meta("FILE", "Attachments");
 ```
 
-* Attach the file to the work order
+* Attach the file to the work order:
 
 By default,
   
 ```java
 att = ats.create(att);
 ```
 
->**note**: For the custom property name for doclinks such as "customdoclink",
+>**Note**: For the custom property name for doclinks such as "customdoclink",
```

Comment:
line 652
This note is missing a verb, so I can't tell what you want the reader to do with this information.